deploy/gke/simple/README.md

```diff
@@ -3,7 +3,11 @@
 If this is the first example you are trying out, follow the [Setup](../setup.md) instructions to
 complete the prerequisites.
 
-See the [Troubleshooting](../troubleshooting.md) guide for more information if you encounter issues.
+**NOTE:** As a reminder, if you completed the pre-requisite steps successfully, the following requirements should have already been met - 
+ - You should see the image name in [manifest.yaml](./manifest.yaml) updated with the fully qualified docker image name. 
+ - The above mentioned docker image is actually uploaded to Artifact Registry and is visible there in cloud console.
+
+If the above requirements are not met, please ensure that all the [Setup](../setup.md) instructions have been followed. You may need to perform a [local build](../../../build/local/README.md) again.
```

Comment:
this feels redundant to the links that are in the setup doc

Reply:
So while trying the steps from this sample, treating `build` and `gke` as 2 different/unrelated things and doing them separately what I failed to realize was that running the local build actually modifies the  `manifest.yaml` file. 

Also, the local build will have to be run for both samples independently (since it only modifies 1 `manifest.yaml` per build). 
So I believe while a little redundant, having this section here makes the process a little more transparent and easier to follow.